src/clear-refinements/clear-refinements.ts

```diff
@@ -28,15 +34,15 @@ export class NgAisClearRefinements extends BaseWidget {
   // instance options
   @Input() public includedAttributes: string[];
   @Input() public excludedAttributes: string[];
-  // TODO: add transformItems
+  @Input() public transformItems?: <U extends string>(items: string[]) => U[];
```

Comment:
What's the benefit of extending `string` here?

Reply:
You're there is not benefit. I'll remove it.

Reply:
fixed here: a4d7f36